### PR TITLE
adding favorite locations to map

### PR DESCRIPTION
**Pull request by hadleyshapland**
Added favorite locations in dropdown menu and added a way for users to enter any address.

Here is what it looks like:
image

### CODE REVIEW

**Review by ghandre — APPROVED** [review]
This looks really good Hadley :)

The styling is consistent, and I like the addition of the dropdown menu; Was that your initiative or was that a suggestion in the walkthroughs?

[hshapland]: (It wouldn't let me reply to this for some reason, so I am just editing my response in) Thanks! It was my idea to try that because I thought it looked too cluttered with all of the buttons so I'm glad you like it :)

File: portfolio/src/main/webapp/js/scripts.js [hunk 1]
```diff
@@ -93,13 +97,34 @@ function deleteComment(id) {
...
+  if(goTo == undefined) {
+    goTo = document.getElementById('address').value;
+  }
+
+  geocoder = new google.maps.Geocoder();
+  geocoder.geocode( { 'address': goTo}, function(results, status) {
+    if (status == 'OK') {
+      map.setCenter(results[0].geometry.location);
+      map.setZoom(15);
+      var marker = new google.maps.Marker({
```
Comment (jerrychangusa) [inline portfolio/src/main/webapp/js/scripts.js]: nit: const

File: portfolio/src/main/webapp/js/scripts.js [hunk 2]
```diff
@@ -1,3 +1,7 @@
+//global variables for map
+let geocoder;
```
Comment (jerrychangusa) [inline portfolio/src/main/webapp/js/scripts.js]: does geocoder need to be global?

**Review by jerrychangusa — APPROVED** [review]
LGTM

File: portfolio/src/main/webapp/js/scripts.js (hunk 1 above)
Comment (hadleyshapland) [reply portfolio/src/main/webapp/js/scripts.js]: Fixed, thank you

File: portfolio/src/main/webapp/js/scripts.js (hunk 2 above)
Comment (hadleyshapland) [reply portfolio/src/main/webapp/js/scripts.js]: Good point, it doesn't need to be. I changed it so that only map is a global variable. Thanks!

**Pull request merged**